src/hmr.ts

```diff
@@ -114,6 +115,12 @@ export async function handleHotUpdate(
     // template is inlined into main, add main module instead
     if (!templateModule) {
       affectedModules.add(mainModule)
+    } else if (mainModule) {
+      for (const importer of mainModule.importers) {
+        if (isCSSRequest(importer.url) && !affectedModules.has(importer)) {
```

Comment:
```suggestion
        if (isCSSRequest(importer.url)) {
```

It's a Set and you don't need to do the existing check

Reply:
fixed